### DISCUSSION TITLE
Call for new code reviewers 📞 💸

### DISCUSSION THREAD

**Discussion by JamieSlome** [418sec/huntr#1343] [Q&A] [answered]
We are now looking for new sheriffs, a.k.a. more legendary code reviewers :crossed_swords:

Our awesome sheriff team includes:

@Mik317 [Profile](https://huntr.dev/users/Mik317), @bbeale [Profile](https://huntr.dev/users/bbeale), @mufeedvh [Profile](https://huntr.dev/users/mufeedvh), @toufik-airane [Profile](https://huntr.dev/users/toufik-airane)

As a sheriff, you will:

- Review vulnerability fixes and disclosures submitted by our community
- Get paid for reviews on successful disclosures and fixes 💵 
- Get a shiny sheriff badge on your profile :star: 
- Join our private Discord channel :detective: 
- Join weekly team meetings for platform updates 📆 
- Help us shape the direction for the platform :railway_track:
- Engage with project maintainers to discuss vulnerability fixes and disclosures :hammer: 

To check out our existing disclosures awaiting feedback, click [here](https://github.com/418sec/huntr/pulls?q=is%3Aopen+is%3Apr+label%3Adisclosure)...

If you are interested and want to get involved, feel free to show your interest in the comments below!

For any interest shown, I will contact you on [Discord](https://huntr.dev/discord) or look to schedule a call with you for the next steps!

Cheers! 🍰

**Comment by B3EF** [comment]
i am interested ...<3

**Reply by JamieSlome** [reply]
Thanks for showing your interest @B3EF - let's schedule a call for next week!

**Reply by B3EF** [reply]
sure <3

**Comment by Asjidkalam** [comment]
Interested!! 🦆❤️

**Reply by JamieSlome** [reply]
@Asjidkalam - hey! Thanks for showing your interest. I will drop you a message on Discord and we can organise next steps?

**Reply by Asjidkalam** [reply]
sure!
anytime..

**Comment by mzfr** [comment]
Hey, I am interested in this

**Reply by JamieSlome** [reply]
Hey @mzfr - thanks for showing your interest! If you have an account on our [platform](https://huntr.dev), I will set up a call with you next week! 🍰 

Also, feel free to join our Discord (link provided above).

**Reply by mzfr** [reply]
@JamieSlome I already had an account on the platform. Now I've also joined the discord.

**Reply by JamieSlome** [reply]
@mzfr - great, I will get in contact with you next week!

**Comment by nikkolai14** [comment]
Hey huntr! I am interested in this role. Thanks

**Reply by JamieSlome** [reply]
Hey @nikkolai14, thanks for showing your interest! I will send you an invite for a call next week!

**Comment by d3m0n-r00t** [comment]
I am interested.

**Reply by JamieSlome** [reply]
@d3m0n-r00t - thanks!

I will shoot over another message to you on Discord next week and we can have a call together!

Have a great weekend! 🍰

**Reply by d3m0n-r00t** [reply]
Sure. 
You too have a nice weekend.

**Comment by b1nslashsh** [comment]
iam interested, Thanks

**Reply by JamieSlome** [reply]
@b1nslashsh - thanks for getting in touch - let's schedule a call for next week! 🍰

**Reply by b1nslashsh** [reply]
sure~

**Comment by toufik-airane** [comment]
Awesome! 👏 🎉

**Comment by ghost** [comment]
Count me in I’m interested as well

**Reply by JamieSlome** [reply]
@galepsoto - awesome! Make sure you have signed up to the [platform](https://huntr.dev) and joined our [Discord](https://huntr.dev/discord).

Once you have done that, I will arrange a call next week! 🍰

**Comment by Anon-Artist** [comment]
I'm interested, Such an awesome opportunity 😍

**Reply by JamieSlome** [reply]
Hey @Anon-Artist - thanks for showing your interest - I will get in touch with you next week to schedule a call! 🤙

**Reply by Anon-Artist** [reply]
Yeah sure at any time and have a nice weekend 😊

**Comment by alromh87** [comment]
:hand:

**Reply by JamieSlome** [reply]
Hey @alromh87 - I saw that you joined our Discord earlier, so I will drop you a message there next week!

**Comment by jibanta66** [comment]
I am interested

**Comment by kajalNair** [comment]
I am interested.

**Comment by JamieSlome** [comment]
Thanks to all for showing their interest! We are really excited about the number of people that got in touch! 🤙 

Due to high demand, if you are still interested, please set up a meeting using my Calendly invite link:

https://calendly.com/jamieslome/code-review-interview?month=2020-12

I look forward to speaking with you soon...

Cheers! 🍰

**Comment by rohan282** [comment]
Interested!

**Reply by JamieSlome** [reply]
@rohan282 - thanks for showing your interest! Feel free to schedule a call using the Calendly link above, cheers! 🍰

**Comment by jibanta66** [comment]
Yes I am

**Comment by JamieSlome** [comment]
FYI, we are currently freezing code reviewer interviews, so the above Calendly link no longer works.

Once we continue, I will enable the link again.

Cheers! 🍰

**Comment by jayateertha043** [comment]
I am interested !!!

**Comment by JamieSlome** [comment, marked as answer]
Thanks for all of the interest shown.

Closing this discussion as it has become stale - cheers! 🍰
